src/auth/components/sideImage/SideImage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+const mainImage = require("../../../shared/assets/images/main-image.jpg");
```

Comment:
change this to import instead of require. consistency.

Reply:
Done.